tests/opd-and-delius/opd-assessment.spec.ts

```diff
@@ -29,10 +29,7 @@ test('OPD assessment creates an event in Delius', async ({ page }) => {
             appearanceType: 'Sentence',
         },
     })
-    await oasysLogin(page, UserType.Booking)
-    // And I select "Warwickshire" from Choose Provider Establishment
-    await selectRegion(page)
```

Comment:
Right now the user OASYS_USERNAME_ASSESSMENT doesn't show the page to select a region. i have asked for the user to show the screen like the other users, when that change is done, i will enable the select region.